components/user/entry-points/user.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { StatusCodes } from "http-status-codes";
+import { submitUserAnswer } from "../domain/user.service";
+import { Request, Response } from "express";
+
+export const submitAnswer = async (req: Request, res: Response) => {
+  try {
+    await submitUserAnswer(req.body);
+    res.status(StatusCodes.CREATED);
```

Comment:
use either sendStatus() or status().send()
https://stackoverflow.com/questions/38620682/difference-between-response-status-vs-response-sendstatus-in-express

Reply:
recheck that comment pls